packages/calcite-components/src/components/flow-item/flow-item.tsx

```diff
@@ -168,7 +168,7 @@ export class FlowItem
   /**
    * Fires when the back button is clicked.
    */
-  @Event({ cancelable: false }) calciteFlowItemBack: EventEmitter<void>;
+  @Event({ cancelable: true }) calciteFlowItemBack: EventEmitter<void>;
```

Comment:
@jcfranco I wonder if we should have a common test helper to test all the custom events on a component to make sure that an event doesn't switch from `cancelable: true` to `cancelable: false` as that would be a breaking change.

I'm thinking kind of like a `defaults` test to make sure events are setup as they should be.

Reply:
This would be a great helper, but would be hard to generalize since we would need to still pass a function that triggers and asserts the canceled behavior. The actual event being canceled is half of what this helper would need to assert.

Reply:
Yeah, might be overkill. Just a thought.